napari/components/add_layers_mixin.py

```diff
@@ -123,8 +124,9 @@ def add_image(
             Threshold for isosurface.
         attenuation : float
             Attenuation rate for attenuated maximum intensity projection.
-        name : str
-            Name of the layer.
+        name : str or list of str
+            Name of the layer.  If a list then must be same length as the axis
+            that is being expanded as channels.
         metadata : dict
             Layer metadata.
         scale : tuple of float
```

Comment:
do we need to add more "or list of ..." to these doc strings?

Reply:
whoops.  def.